xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1835 @@
+              - horizon
+              - heat-api
+              - heat-api-cfn
+ -            - heat-api-cloudwatch
+              - heat-engine
+              - ops-console-web
+              - barbican-api
+ @@ -151,7 +147,6 @@
+              - magnum-api
+              - magnum-conductor
+              - manila-api
+ -            - manila-share
+
+          - name: mml
+            cluster-prefix: mml
+ @@ -164,9 +159,7 @@
+
+              # freezer-api shares elastic-search with logging-server
+              # so must be co-located with it
+ -            - freezer-api
+
+ -            - ceilometer-api
+              - ceilometer-polling
+              - ceilometer-agent-notification
+              - ceilometer-common
+ @@ -194,4 +187,3 @@
+              - neutron-l3-agent
+              - neutron-metadata-agent
+              - neutron-openvswitch-agent
+ -            - neutron-lbaasv2-agent
+      </screen>
+    </step>
+    <step>
+      <para>
+        If you are happy with these changes, commit them into
+        the Git repository as follows:
+      </para>
+      <screen>&prompt.ardana;cd ~/openstack/my_cloud/definition
+ &prompt.ardana;git add -A
+ &prompt.ardana;git commit -m "SOC 9 CLM Upgrade input model migration"
+      </screen>
+    </step>
+    <step>
+      <para>
+        Now you are ready to run the <filename>config-processor-run.yml</filename> playbook. If the
+        necessary input model changes have been made, it will complete
+        sucessfully:
+      </para>
+      <screen>&prompt.ardana;cd ~/openstack/ardana/ansible
+ &prompt.ardana;ansible-playbook -i hosts/localhost config-processor-run.yml
+ Enter encryption key (press return for none):
+ confirm Enter encryption key (press return for none):
+ To change encryption key enter new key (press return for none):
+ confirm To change encryption key enter new key (press return for none):
+
+ PLAY [localhost] **************************************************************
+
+ GATHERING FACTS ***************************************************************
+ ok: [localhost]
+
+ ...
+ PLAY RECAP ********************************************************************
+ localhost                  : ok=24   changed=20   unreachable=0    failed=0
+     </screen>
+   </step>
+ </procedure>
+ </section>
+
+ <section xml:id="upgrade-cloud-services">
+   <title>Upgrading Cloud Services</title>
+   <para>
+     The deployer node is now ready to be used to upgrade the remaining
+     cloud nodes and running services.
+   </para>
+   <note>
+     <para>
+       The <filename>ardana-upgrade.yml</filename> playbook runs the upgrade process
+       against all nodes in parallel, though some of the steps are serialised
+       to run on only one node at a time to avoid triggering potentially
+       problematic race conditions. As such, the playbook can take a long time to run.
+     </para>
+   </note>
+
+   <procedure>
+     <title>Generate the Based Scratch Area</title>
```

Comment:
Should this not be $something-based not just Based?

Reply:
Checked in @rtamalin's original

https://confluence.suse.com/display/~fmccarthy/SOC+9+CLM+Upgrade+Documentation

... where it says:

> Generate the SOC 9 CLM Based Scratch Area